pom.xml

```diff
@@ -79,6 +79,14 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <!-- required for declarative pipeline tests -->
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-aggregator</artifactId>
+            <version>2.7</version>
```

Comment:
I was wondering why you had to provide the version here. It seems that it is not provided in the BOM.
See the workflow-aggregator [README](https://github.com/jenkinsci/workflow-aggregator-plugin/commit/b59db644e1e7eb2ebf16848a6c7a46c8a65cd7b1) what should be used as dependency.

Reply:
> I was wondering why you had to provide the version here. It seems that it is not provided in the BOM.

I guess this is because plugins should not depend on the workflow-aggregator.